modules/wms/src/lib/parsers/wms/parse-wms-capabilities.ts

```diff
@@ -143,7 +143,7 @@ export type WMSExceptions = {
   mimeTypes: string[];
 };
 
-export type parseWMSCapabilitiesOptions = {
+export type ParseWMSCapabilitiesOptions = {
   /** Add inherited layer information to sub layers */
   inheritedLayerProps?: boolean;
```

Comment:
By setting this prop, loaders.gl will inject inherited props on all child layers. It does make the resulting JSON massive.